src/main/java/org/stellar/sdk/responses/TransactionResponse.java

```diff
@@ -164,6 +171,41 @@ public Links getLinks() {
     return links;
   }
 
+  /**
+   * Preconditions of a transaction per <a href="https://github.com/stellar/stellar-protocol/blob/master/core/cap-0021.md#specification">CAP-21<a/>
+   */
+  @Value
+  public static class Preconditions {
+    @SerializedName("timebounds")
+    TimeBounds timeBounds;
+    @SerializedName("ledgerbounds")
+    LedgerBounds ledgerBounds;
+    @SerializedName("min_account_sequence")
+    long minAccountSequence;
```

Comment:
There is a distinction between when `minAccountSequence` is set or not, so you can't treat not set to a default value like 0. Not set means one-less than the tx seq, but 0 means 0. @Shaptic This may be an issue in Horizon's API and in other SDKs.

Reply:
See related conversation: https://discord.com/channels/897514728459468821/971454936615358485/971454986988957736

Reply:
ah, ok, thanks for mentioning this, I updated the sdk here to model that as a nullable object to signal if wasn't present in the json.

Reply:
It's a pointer in the API, so we're good